src/backports/configparser/__init__.py

```diff
@@ -684,7 +684,7 @@ def read(self, filenames, encoding=None):
 
         Return list of successfully read files.
         """
-        if PY2 and isinstance(filenames, bytes):
+        if PY2 and isinstance(filenames, bytes) and sys.py3kwarning:
```

Comment:
This additional guard should have been added inside the `if`, only for the `warnings.warn()`.
By adding it here, `filenames = [filenames]` is not executed.

This is breaking [flake8](http://flake8.pycqa.org) (which depends on configparser under Python 2) for me, `setup.cfg` of my project seems to be ignored with this change, while it works normally if I remove it.

I see that this code has just been removed in the master branch, are you planning to release configparser 3.7.0 soon?

Reply:
Which causes #27

Reply:
> are you planning to release configparser 3.7.0 soon?

Yes. I actually thought I had. Indeed, I'd tagged it, just not pushed the releases. I'll do that now.